x-pack/plugins/cloud_security_posture/public/common/constants.ts

```diff
@@ -6,8 +6,44 @@
  */
 
 import { euiPaletteForStatus } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { CLOUDBEAT_EKS, CLOUDBEAT_VANILLA } from '../../common/constants';
 
 const [success, warning, danger] = euiPaletteForStatus(3);
 
 export const statusColors = { success, warning, danger };
 export const CSP_MOMENT_FORMAT = 'MMMM D, YYYY @ HH:mm:ss.SSS';
+
+export type CloudPostureIntegrations = typeof cloudPostureIntegrations;
+
+export const cloudPostureIntegrations = {
+  kspm: {
+    policyTemplate: 'kspm',
+    name: i18n.translate('xpack.csp.kspmIntegration.integration.nameTitle', {
+      defaultMessage: 'Kubernetes Security Posture Management',
+    }),
+    shortName: i18n.translate('xpack.csp.kspmIntegration.integration.shortNameTitle', {
+      defaultMessage: 'KSPM',
+    }),
+    options: [
+      {
+        type: CLOUDBEAT_VANILLA,
+        name: i18n.translate('xpack.csp.kspmIntegration.vanillaOption.nameTitle', {
+          defaultMessage: 'Unmanaged Kubernetes',
+        }),
+        benchmark: i18n.translate('xpack.csp.kspmIntegration.vanillaOption.benchmarkTitle', {
+          defaultMessage: 'CIS Kubernetes',
+        }),
+      },
+      {
+        type: CLOUDBEAT_EKS,
+        name: i18n.translate('xpack.csp.kspmIntegration.eksOption.nameTitle', {
+          defaultMessage: 'EKS (Elastic Kubernetes Service)',
+        }),
+        benchmark: i18n.translate('xpack.csp.kspmIntegration.eksOption.benchmarkTitle', {
+          defaultMessage: 'CIS EKS',
+        }),
```

Comment:
`name` and `benchmark` are very similar. did we not manage to combine them?

Reply:
its been up for discussion, the main argument is that maybe in the future we will feature deployments with multiple benchmarks. id still argue that we can remove the deployment type layer and just select and display multiple benchmarks. for now, i wanted to do what was planned by product, hopefully we will revisit this soon